light-client/tests/support

```diff
@@ -1 +0,0 @@
-../../tendermint/tests/support/lite
```

Comment:
Just to double check: these tests are called from their new location now, right?

Reply:
Ah I see, they were always called from the new location. Now they are actually moved there (instead of using the sim-link). All good.

Reply:
Yep, that's it!